lang/tags/keywords.py

```diff
@@ -20,5 +20,13 @@ def code_keyword(m) -> str:
 class Actions:
     def code_keyword(keywords: list[str]):
         """Adds keywords"""
-        for keyword in keywords:
-            actions.insert(keyword)
+        if len(keywords) == 1:
+            actions.insert(keywords[0])
+        else:
+            # every keyword is separated by a space
+            # the spacing before the first keyword and after the last keyword is kept
+            combined_text: str = keywords[0].rstrip() + " "
+            for i in range(1, len(keywords) - 1):
+                combined_text += keywords[i].strip() + " "
```

Comment:
String concatenation in a loop like this is O(n²). This really should use `" ".join`.

Reply:
I am aware that string concatenation in a loop like that is quadratic in python. I was not concerned with the performance given how small the input sizes are going to be in real use cases, but I can make the change if the maintainers want.

Reply:
No one else showed up at today's backlog session to weigh in, so I decided to make the change.